src/main/java/org/tron/core/net/node/TrxHandler.java

```diff
@@ -26,7 +26,7 @@
 
   private NodeImpl nodeImpl;
 
-  private static int MAX_TRX_SIZE = 100_000;
+  private static int MAX_TRX_SIZE = 50_000;
 
   private static int MAX_SMART_CONTRACT_SUBMIT_SIZE = 100;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this field "MAX_SMART_CONTRACT_SUBMIT_SIZE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. ([squid:S3008](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS3008))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1672&issues=AWbTi2U2LpqqSUlcbPrx&open=AWbTi2U2LpqqSUlcbPrx)

[ISSUE_KEY]: AWbTi2U2LpqqSUlcbPrx